franka_gazebo/src/franka_gripper_sim.cpp

```diff
@@ -422,21 +422,33 @@ void FrankaGripperSim::onGraspGoal(const franka_gripper::GraspGoalConstPtr& goal
 
 void FrankaGripperSim::onGripperActionGoal(const control_msgs::GripperCommandGoalConstPtr& goal) {
   ROS_INFO_STREAM_NAMED("FrankaGripperSim", "New Gripper Command Action Goal received: "
+                                                << goal->command.position * 2.0 << "m, "
                                                 << goal->command.max_effort << "N");
 
+  control_msgs::GripperCommandResult result;
+
   // HACK: As one gripper finger is <mimic>, MoveIt!'s trajectory execution manager
   // only sends us the width of one finger. Multiply by 2 to get the intended width.
   double width = this->finger1_.getPosition() + this->finger2_.getPosition();
+  double width_d = goal->command.position * 2.0;
```

Comment:
If you move this line to the top of the function, you can safe the recalculation in line 425